packages/engine/src/renderer/components/HighlightComponent.ts

```diff
@@ -23,6 +23,46 @@ All portions of the code written by the Ethereal Engine team are Copyright © 20
 Ethereal Engine. All Rights Reserved.
 */
 
-import { defineComponent } from '../../ecs/functions/ComponentFunctions'
+import { useEffect } from 'react'
+import { Object3D } from 'three'
 
-export const HighlightComponent = defineComponent({ name: 'HighlightComponent' })
+import { getMutableState, useHookstate } from '@etherealengine/hyperflux'
+
+import { defineComponent, useOptionalComponent } from '../../ecs/functions/ComponentFunctions'
+import { useEntityContext } from '../../ecs/functions/EntityFunctions'
+import { GroupComponent } from '../../scene/components/GroupComponent'
+import { RendererState } from '../RendererState'
+import { EngineRenderer, PostProcessingSettingsState } from '../WebGLRendererSystem'
+
+export const HighlightComponent = defineComponent({
+  name: 'HighlightComponent',
+
+  reactor: function () {
+    const entity = useEntityContext()
+
+    const postProcessingSettingsState = useHookstate(getMutableState(PostProcessingSettingsState))
+    const usePostProcessing = useHookstate(getMutableState(RendererState).usePostProcessing)
+    const group = useOptionalComponent(entity, GroupComponent)
```

Comment:
Just make this `useGroupComponent`, and remove the `if (!group?.value)`  check below